.github/workflows/test.yaml

```diff
@@ -71,7 +71,7 @@ jobs:
 
             - id: components
               run: |
-                components=$(tree src -J -d -L 1 | jq -c '.[0].contents | map(.name)')
+                components=$(find src/ -mindepth 2 -type f -name composer.json -not -path "*/vendor/*" -printf '%h\n' | jq -R -s -c 'split("\n")[:-1] | map(. | sub("^src/";"")) | sort')
```

Comment:
Not sure we should really consider them as "component" :|

Reply:
IDK, I've just added support for Bridges (or more generally, folder with a composer.json inside)